lua/lspconfig/server_configurations/nelua_lsp.lua

```diff
@@ -0,0 +1,39 @@
+local util = require 'lspconfig.util'
+local path = '/usr/local/lib/nelua/lib/nelua-lsp'
```

Comment:
why hardcorde this path. if you download this into other dir .what happen and this path not exist on windows.

Reply:
this is the path where nelua installs and looks for its libraries, only linux of course...
nelua recommends and supports MSYS2 on windows this looks messy

i think making like omnisharp does and letting the user set the cmd is the best i can do, i have no idea of how this would work on windows

Reply:
it's better use system check on this path